src/gpu/intel/ocl/simple_binary.cpp

```diff
@@ -111,6 +120,8 @@ status_t simple_binary_t::pd_t::init_kernel_ctx(
         compute::kernel_ctx_t &kernel_ctx) const {
     def_binary_alg_kinds(kernel_ctx);
     kernel_ctx.define_int("BINARY_ALG", conf.alg);
+    kernel_ctx.define_int(
+            "IS_SELECT_BINARY", (conf.alg == dnnl_binary_select) ? 1 : 0);
```

Comment:
Here and everywhere.
```suggestion
            "IS_SELECT_BINARY", (conf.alg == alg_kind::binary_select) ? 1 : 0);
```